src/Framework/Assert.php

```diff
@@ -2269,6 +2269,12 @@ public static function assertJsonFileNotEqualsJsonFile($expectedFile, $actualFil
         static::assertThat($actualJson, new LogicalNot($constraintExpected), $message);
     }
 
+    public static function assertResourceType($expectedType, $actual, $message = '')
+    {
+        static::assertThat(is_resource($actual), new IsTrue(), $message);
```

Comment:
It would be better to use `assertInternalType()` here.

Reply:
Actually, it would be better to implement a proper `Constraint` object.

Reply:
Thanks for feedback, `IsResource` constraint implemented.